projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.component.html

```diff
@@ -1,3 +1,4 @@
+<ng-content select="igx-grid-state,igc-grid-state"></ng-content>
```

Comment:
Don't think the pivot had an actual sink container till now (one of the outlets is marked to keep the query working) since it didn't really have columns or other elements to project. So might be time to create that hidden div here too